### PR TITLE
Deps update 20210702

### PR DESCRIPTION
**Pull request by alexcjohnson**

### CODE REVIEW

File: tests/selenium/test_scrolling.py [hunk 1]
```diff
@@ -2,11 +2,12 @@
 import dash.testing.wait as wait
 from dash_table import DataTable
 
+import os
```
Comment (eff-kay) [inline tests/selenium/test_scrolling.py:5]: its probably better not to import entire modules, they pollute the namespace. Import the relevant stuff. i.e. 

`from os.path import join, dirname`

**Comment by eff-kay** [comment]
LGTM.

File: tests/selenium/test_scrolling.py (hunk 1 above)
Comment (alexcjohnson) [reply tests/selenium/test_scrolling.py:5]: Not critical, as you say, but I think I disagree about this actually: `import os` adds one thing to the namespace, and the same every time, whereas `from os.path import …` adds N. Plus in usage `os.path.join` is very clear, whereas `join` by itself could be confused for string join or at least make you look back to the top to see which join it is.

Comment (eff-kay) [reply tests/selenium/test_scrolling.py:5]: yea makes sense.

**Pull request merged**